lucille-core/src/main/java/com/kmwllc/lucille/stage/AddRandomField.java

```diff
@@ -0,0 +1,155 @@
+package com.kmwllc.lucille.stage;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.kmwllc.lucille.core.ConfigUtils;
+import com.kmwllc.lucille.core.Document;
+import com.kmwllc.lucille.core.Stage;
+import com.kmwllc.lucille.core.StageException;
+import com.kmwllc.lucille.util.FileUtils;
+import com.typesafe.config.Config;
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+
+/**
+ * Fetches byte data of a given URL field and places data into a specified field
```

Comment:
this line seems to have been copied from a different stage and doesn't apply here

Reply:
done